docs/src/main/paradox/akka-management.md

```diff
@@ -255,3 +230,26 @@ Java
     httpClusterManagement.stop();
     ```
 
+## Developing Extensions
+
+This project provides a set of management extensions. If you want to write third-party extensions to Akka management here
+are few pointers about how it all works together.
+
+The `akka-management` module provides the central HTTP endpoint to which extensions can register themselves.
+
+For example, the "Cluster HTTP Management" module exposes HTTP routes that can be used to monitor,
+and even trigger joining/leaving/downing decisions via HTTP calls to these routes. The routes and
+logic for these are implemented inside the `akka-management-cluster-http`.
+
+An extension can contribute to the exposed HTTP routes by appending to the `akka.management.http.route-providers` list in
+its own `reference.conf` (make sure to use `+=` instead of `=`). The core `AkkaManagement` extension
+then collects all the routes and serves them together under the management HTTP server. This is in order
```

Comment:
```management``` vs ```Management```? I don't know, just bringing it up

Reply:
Yeah, good point. I guess we are a bit sloppy on deciding on consistent terms and following through with them.

Reply:
Added https://github.com/lightbend/paradox/issues/274 to add a term/spell checker to paradox (though, that wouldn't have helped here I guess).